introduction.md

```diff
@@ -0,0 +1 @@
+Hello, this repository has been forked from Deadly Vipers for the Course Mastering Github on CodeSchool
```

Comment:
Looks good! Could you also please mention your favorite Code School path in your introduction? This could be Ruby, JavaScript, HTML & CSS, iOS or Electives.